llvm/test/CodeGen/RISCV/rvv/vp-splat.ll

```diff
@@ -273,247 +275,253 @@ define <vscale x 8 x i64> @vp_splat_nxv8i64(i64 %val, <vscale x 8 x i1> %m, i32
 }
 
 define <vscale x 1 x bfloat> @vp_splat_nxv1bf16(bfloat %val, <vscale x 1 x i1> %m, i32 zeroext %evl) {
-; CHECK-LABEL: vp_splat_nxv1bf16:
-; CHECK:       # %bb.0:
-; CHECK-NEXT:    fmv.x.w a1, fa0
-; CHECK-NEXT:    vsetvli zero, a0, e16, mf4, ta, ma
-; CHECK-NEXT:    vmv.v.x v8, a1
-; CHECK-NEXT:    ret
+; NOZFMIN-LABEL: vp_splat_nxv1bf16:
```

Comment:
Missing an H in the check prefix name

Reply:
I wrote ZFMIN since it has both zfhmin and zfbfmin. Would it be better to be explicit and use ZFHMINZFBFMIN?

Reply:
In that case. I don't have a problemw with keeping it as ZFMIN